config/locales/en.yml

```diff
@@ -90,6 +85,12 @@ en:
       policies: "Policies"
       statistical_data_sets: "Statistical data sets"
       topical_events: "Topical events"
+    show_password:
```

Comment:
I'd suggest adding a comment here so that people in the future know that `show` and `hide` should only be used with the show password component - maybe with a link to this pull request to explain in further detail the why.